test/e2e/network/service.go

```diff
@@ -1183,7 +1183,7 @@ var _ = SIGDescribe("Services", func() {
 		err = t.DeleteService(serviceName)
 		Expect(err).NotTo(HaveOccurred())
 
-		hostExec := framework.LaunchHostExecPod(f.ClientSet, f.Namespace.Name, "hostexec")
+		hostExec := framework.LaunchHostExecPod(f.ClientSet, f.Namespace.Name, "hostexec", false)
```

Comment:
How would this work without hostNet?  It expects `ss` to be able to run and see kube-proxy